ckan/requirements.txt

```diff
@@ -23,7 +23,7 @@ ckanext-envvars==0.0.1
 -e git+https://github.com/ckan/ckanext-qa.git@d7d384cb18a85243ea62ef0bcc76bc1775f1c806#egg=ckanext-qa
 -e git+https://github.com/davidread/ckanext-report.git@2abba3fa2ddf29f4decc4a0c4c39101fb994db23#egg=ckanext-report
 -e git+https://github.com/keitaroinc/ckanext-saml2auth.git@d77d85349c7a6fcfa5a14c19a56b53eacf137a7c#egg=ckanext-saml2auth
--e git+https://github.com/ckan/ckanext-spatial.git@5a987828cb8e76f4904c6f23542a593255171a94#egg=ckanext-spatial
+-e git+https://github.com/ckan/ckanext-spatial.git@4ac25f19aa4eb9c798451f5eeb3084f907ccc003#egg=ckanext-spatial
```

Comment:
Is this newer or older?

Reply:
Old known working version. New one does not work.